lib/health.js

```diff
@@ -48,17 +48,37 @@ class HealthCheck {
     };
   }  
 
+  http() {
+    return (request, response) => {
+      const requestPath = request.url;
+      if (requestPath !== this.getHealthUri()) return false;
+
+      const self = this;
+
+      (async function () {
```

Comment:
if this was `(async () => {` you wouldn't need to redeclare this to self.

Reply:
Touche! Fixed